### DISCUSSION TITLE
[Function Returned Unexpected Results]

### DISCUSSION THREAD

**Discussion by uguraltinsoy** [LupeCode/phpTraderNative#15] [Help]
### Function Name:
Trader

### Input Variables:
Arrays

### Expected Results:
RSI

### Actual Results:
*What results did you actually receive?*

### Comparison Source:

<img width="929" alt="Ekran Resmi 2022-09-28 01 09 06" src="https://user-images.githubusercontent.com/48494791/192646679-b1c2dabc-920b-4c65-99f4-6f745be56e6d.png">

**Comment by jb-lopez** [comment]
I'm going to need more details because it looks like you have a few bugs in your code from that screenshot.

**Comment by uguraltinsoy** [comment]
```php
<?php

namespace LupeCode\phpTraderNative;
use LupeCode\phpTraderNative\Trader;

require_once __DIR__.'/vendor/autoload.php';

$close_value = array();
$rsi = Trader::rsi($close_value, 14);
echo "RSI >> $rsi";
```

**Comment by jb-lopez** [comment]
You don't want to set your namespace to `LupeCode\phpTraderNative`, your projects namespace will be something different. And you cannot echo the results like that. An array cannot be implicitly converted to a string in PHP. Also you are passing in an empty array. There is nothing to calculate.

I'm not sure what you expect this code to do.
